operator/v1/types_dns.go

```diff
@@ -376,6 +413,21 @@ const (
 	NetworkResolverType   UpstreamType = "Network"
 )
 
+// ProtocolStrategy is a preference for the protocol to use for DNS queries.
```

Comment:
What should a consumer of this do if there's a value that they do not recognise? Think about what you'd expect the operator to do if a new value was added to this enum?

Reply:
Got it, done. Not sure if you literally wanted `---` and a `+`, but let me know if you did.

Reply:
Yeah I did 😅 Sorry, should have explained, the format I put there means that the doc comment will only be visible here, it prevents the doc being picked up in any swagger or openapi copies of the docs. `---` prevents swagger taking anything after it, `+` stops openapi picking it up

I was wrong though, the `---` line needs to be `// + ---`

Reply:
Interesting okay. I pushed up again, let me know if I interpreted you correctly.

Reply:
Yes, perfect, thanks